reg_tests/global_cycle/C192.jedi_lndincsoilnoahmp.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+#------------------------------------------------------------------
+# Run global_cycle for a C192 case to test the ingest and
+# application of soil moisture and temperature increments
+# on the cubed-sphere grid into Noah-MP restarts, which should
+# yield identical results as compared with the GSI case.
+# Compare output to a baseline set of files using the 'nccmp' 
+# utility.
+#------------------------------------------------------------------
+
+set -x
+
+NCCMP=${NCCMP:-$(which nccmp)}
+
+export MAX_TASKS_CY=6
+
+export HOMEgfs=$NWPROD
+export BASE_GSM=$NWPROD
+
+export CYCLEXEC=$BASE_GSM/exec/global_cycle
+
+export CDATE=2019073000
+export FHOUR=00
+export DELTSFC=6
+
+export CASE=C192
+export OCNRES=99999
+
+export COMIN=$HOMEreg/input_data_noahmp
+
+export JCAP=1534
+export LONB=3072
+export LATB=1536
+
+export OROFIX=$HOMEreg/fix/$CASE
+export FIXgsm=$BASE_GSM/fix/am
+
+export DONST="NO"
+export use_ufo=.true.
+
+export DO_SFCCYCLE=".FALSE." 
+export DO_LNDINC=".TRUE." 
+export DO_SOI_INC_JEDI=".true."
+
+export VERBOSE=YES
+export CYCLVARS=FSNOL=-2.,FSNOS=99999.,
+
+$BASE_GSM/ush/global_cycle_driver.sh
+
+iret=$?
+if [ $iret -ne 0 ]; then
+  set +x
+  echo "<<< C192 JEDI based LANDINC SOIL NOAHMP CYCLE TEST FAILED. >>>"
+  exit $iret
+fi
+
+test_failed=0
+
+cd $DATA
+for files in *tile*.nc
+do
+  if [ -f $files ]; then
+    echo CHECK $files
```

Comment:
You use the same baseline as the C192.gsi_lndincsoilnoahmp test? Is that intentional?

Reply:
Yes, for now, I just intentionally set the reference as the old baseline for both regression tests. Once the PR is OK, will set up with new baselines.

Reply:
I created separate baselines for testing on Hera. 

```
/scratch1/NCEPDEV/nems/role.ufsutils/ufs_utils/reg_tests/global_cycle/baseline_data $ ls -l
total 68
lrwxrwxrwx 1 role.ufsutils nems   30 Feb 21 21:01 c192.gsi_lndincsoilnoahmp -> c192.gsi_lndincsoilnoahmp.1234
dr-xr-sr-x 2 role.ufsutils nems 4096 Feb 21 21:00 c192.gsi_lndincsoilnoahmp.1234
lrwxrwxrwx 1 role.ufsutils nems   31 Feb 21 21:01 c192.jedi_lndincsoilnoahmp -> c192.jedi_lndincsoilnoahmp.1234
dr-xr-sr-x 2 role.ufsutils nems 4096 Feb 21 21:01 c192.jedi_lndincsoilnoahmp.1234
```

The '1234' is a placeholder for the future commit hash. Each baseline is just a copy of the `c192lndincsoilnoahmp` baseline directory.

Now you can adjust the scripts as follows:

```
--- a/reg_tests/global_cycle/C192.gsi_lndincsoilnoahmp.sh
+++ b/reg_tests/global_cycle/C192.gsi_lndincsoilnoahmp.sh
@@ -62,7 +62,7 @@ for files in *tile*.nc
 do
   if [ -f $files ]; then
     echo CHECK $files
-    $NCCMP -dmfqS $files $HOMEreg/baseline_data/c192.lndincsoilnoahmp/$files
+    $NCCMP -dmfqS $files $HOMEreg/baseline_data/c192.gsi_lndincsoilnoahmp/$files
     iret=$?
```

```
--- a/reg_tests/global_cycle/C192.jedi_lndincsoilnoahmp.sh
+++ b/reg_tests/global_cycle/C192.jedi_lndincsoilnoahmp.sh
@@ -62,7 +62,7 @@ for files in *tile*.nc
 do
   if [ -f $files ]; then
     echo CHECK $files
-    $NCCMP -dmfqS $files $HOMEreg/baseline_data/c192.lndincsoilnoahmp/$files
+    $NCCMP -dmfqS $files $HOMEreg/baseline_data/c192.jedi_lndincsoilnoahmp/$files
     iret=$?

```

Reply:
Done. Thanks!